packages/mdc-switch/_mixins.scss

```diff
@@ -289,10 +300,6 @@
       functions.transition(background-color),
       functions.transition(border-color);
   }
-
-  @include feature-targeting-mixins.targets($feat-color) {
```

Comment:
Hey Andrew, would a possible fix for this be to remove the color feature-target here? Color is an inaccurate feature for this anyway, it's more-so structure.

Reply:
My main thinking is the border for the track isn't designed to be themed in Material and the transparent border-color is purely for high-contrast mode, not any actual theming purposes.

Reply:
SGTM - I agree with the notion that it's structure (at least when compared to color or the other features)